test/helper.rb

```diff
@@ -39,6 +39,8 @@
 require "mocha/setup"
 require "net/http"
 
+require 'openssl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.